policyengine_us/variables/gov/states/de/tax/income/subtractions/pension_exclusion/de_pension_exclusion.py

```diff
@@ -0,0 +1,83 @@
+from policyengine_us.model_api import *
+
+
+class de_pension_exclusion(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Delaware pension exclusion"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://revenuefiles.delaware.gov/2022/PIT-RES_TY22_2022-02_Instructions.pdf#page=6",
+        "https://delcode.delaware.gov/title30/c011/sc02/index.html",
+    )
+    defined_for = StateCode.DE
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.de.tax.income.subtractions.exclusions.pension
+
+        person = tax_unit.members
+
+        # determine age eligibility
+        age = person("age", period)
+        head = person("is_tax_unit_head", period)
+        spouse = person("is_tax_unit_head", period)
+
+        head_or_spouse = head | spouse
+        age_head_or_spouse = age * head_or_spouse
```

Comment:
Don't need - just use `age` and then multiply by `head_or_spouse` at the end if that's the eligibility criterion